101-1vm-2nics-2subnets-1vnet/azuredeploy.parameters.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
can you fix the tool to not write this file if you need no changes to it?

Reply:
I updated it and tried to revert, but there are still whitespace differences.

Reply:
Ok, now it's fixed.